kivymd/tools/patterns/create_project.py

```diff
@@ -589,16 +589,16 @@ def generate_application_screens(self) -> None:
 .PHONY: po mo
 
 po:
-	xgettext -Lpython --output=messages.pot --from-code=utf-8 \\
```

Comment:
Why do you use 8 spaces instead of 4?

Reply:
I used 4 spaces, the line you pointed is a tab (old) and the line below is using 4-spaces (new)